app/cypress/fixtures/documentTests.js

```diff
@@ -1,8 +1,8 @@
-import minimalDoc from '../shared/minimalCSAFBaseDoc'
-import minimalInformationalAdvisoryDoc from '../shared/minimalInformationalAdvisoryDoc'
-import minimalSecurityAdvisoryDoc from '../shared/minimalSecurityAdvisoryDoc'
-import minimalSecurityIncidentResponseDoc from '../shared/minimalSecurityIncidentResponseDoc'
-import minimalVexDoc from '../shared/minimalVexDoc'
+import minimalDoc from './documentTests/minimalGenericCSAFDoc'
```

Comment:
I guess that must be `minimalCSAFBaseDoc`. See #234 

´´´suggestion
import minimalDoc from './documentTests/minimalCSAFBaseDoc'

Reply:
Done in [3e5687f](https://github.com/secvisogram/secvisogram/pull/241/commits/3e5687f006764a7938f8d747c8d860d47e732920).